_posts/2019-03-26-read-models-a-knot-to-untie.md

```diff
@@ -53,7 +53,7 @@ class SampleHandler : IHandleRequests
 }
 ```
 
-the infrastructure executing the above code could, at runtime, store the resulting ViewModel, the `dynamic vm`, in a dedicated incredibly fast storage, and query that storage for all the subsequent requests for the same URL. We've basically created a read model at runtime at the first request. So far, so good.
+the infrastructure executing the above code could, at runtime, store the resulting ViewModel, the `dynamic vm`, in a dedicated incredibly fast storage, and query that storage for all subsequent requests for the same URL. We've basically created a read model at runtime at the first request. So far, so good.
 
 ### You know what?
 
```

Comment:
Maybe consider a more descriptive title than "Ownership dilemma". It's not immediately obvious in the transition that you're about to talk about the problems with caching.

Reply:
Changed